src/main/webapp/WEB-INF/web.xml

```diff
@@ -65,4 +70,10 @@
     <url-pattern>/stats</url-pattern>
     <servlet-name>stats</servlet-name>
   </servlet-mapping>
+
+  <servlet-mapping>
+    <url-pattern>/messagechart</url-pattern>
```

Comment:
I'd recommend `/api/chart` (with `/chart` being the URL for the corresponding *.html code).